### PR TITLE
Setup ModelAdmin class for each model in Hosts app

### PR DESCRIPTION
**Pull request by gauthamzz**

### CODE REVIEW

File: apps/hosts/admin.py [hunk 1]
```diff
@@ -1,3 +1,19 @@
 from django.contrib import admin
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import ChallengeHostTeam, ChallengeHost
+
+
+@admin.register(ChallengeHostTeam)
+class HostTeamModelAdmin(TimeStampedAdmin):
```
Comment (TroJan) [inline apps/hosts/admin.py]: class name should be ChallengeHostTeamAdmin. convention ideally is name of model + 'Admin'

File: apps/hosts/admin.py [hunk 2]
```diff
@@ -1,3 +1,19 @@
 from django.contrib import admin
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import ChallengeHostTeam, ChallengeHost
```
Comment (TroJan) [inline apps/hosts/admin.py]: lets write clean code. keep model imports alphabetically. also use tuple like

```
from .models import (ChallengeHost,
                     ChallengeHostTeam,)
```

File: apps/hosts/admin.py [hunk 3]
```diff
@@ -1,3 +1,19 @@
...
+
+@admin.register(ChallengeHostTeam)
+class HostTeamModelAdmin(TimeStampedAdmin):
+    list_display = ("team_name", "created_by")
+    list_filter = ("team_name", "created_by")
+    search_fields = ("team_name", "created_by")
+
+
+@admin.register(ChallengeHost)
+class HostModelAdmin(TimeStampedAdmin):
```
Comment (TroJan) [inline apps/hosts/admin.py]: update class name here also.

**Comment by TroJan** [comment]
@gauthamzz : you need to properly rebase your branch from master. Let me know if you face any issue

**Comment by gauthamzz** [comment]
@TroJan  i hope its okay now ?

File: apps/hosts/admin.py [hunk 4]
```diff
@@ -1,3 +1,20 @@
 from django.contrib import admin
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import (ChallengeHostAdmin,
```
Comment (TroJan) [inline apps/hosts/admin.py]: wrong import. model name concerned here are `ChallengeHostTeam` and `ChallengeHost`

File: apps/hosts/admin.py [hunk 5]
```diff
@@ -1,3 +1,20 @@
...
+                     ChallengeHostTeamAdmin)
+
+
+@admin.register(ChallengeHostTeamAdmin)
+class HostTeamModelAdmin(TimeStampedAdmin):
```
Comment (TroJan) [inline apps/hosts/admin.py]: can you please rename this class to ChallengeHostTeamAdmin

File: apps/hosts/admin.py [hunk 6]
```diff
@@ -1,3 +1,20 @@
...
+    list_display = ("team_name", "created_by")
+    list_filter = ("team_name", "created_by")
```
Comment (TroJan) [inline apps/hosts/admin.py]: can you please remove `team_name` from `list_filter`

File: apps/hosts/admin.py [hunk 7]
```diff
@@ -1,3 +1,20 @@
...
+    search_fields = ("team_name", "created_by")
+
+
+@admin.register(ChallengeHostAdmin)
+class HostModelAdmin(TimeStampedAdmin):
```
Comment (TroJan) [inline apps/hosts/admin.py]: please rename this class to `ChallengeHostAdmin`

File: apps/hosts/admin.py [hunk 8]
```diff
@@ -1,3 +1,20 @@
...
+    list_display = ("user", "team_name", "status")
```
Comment (TroJan) [inline apps/hosts/admin.py]: permissions can be added in list_display

File: apps/hosts/admin.py [hunk 9]
```diff
@@ -1,3 +1,20 @@
...
+    list_filter = ("user", "team_name", "status")
```
Comment (TroJan) [inline apps/hosts/admin.py]: lets give search by `user` and `team_name` and filter by `status` and `permissios`. can you please do these changes?

**Comment by gauthamzz** [comment]
@TroJan please review

**Comment by TroJan** [comment]
@gauthamzz thanks, will review it in night

File: apps/hosts/admin.py (hunk 3 above)
Comment (deshraj) [inline apps/hosts/admin.py]: @gauthamzz Can you replace the class name with `ChallengeHostAdmin` ?

**Comment by deshraj** [comment]
@gauthamzz Let me know if you are working on it. We would like to get it done soon.

**Comment by gauthamzz** [comment]
I have send pr Four days go . @TroJan didnt review it yet for both #150 and #149

**Comment by deshraj** [comment]
@gauthamzz Please check the comments above. There are some changes that need to be done so that we can merge it into master.

File: apps/hosts/admin.py (hunk 3 above)
Comment (gauthamzz) [inline apps/hosts/admin.py]: sorry i didnt see this :-1: one minute

File: apps/hosts/admin.py [hunk 10]
```diff
@@ -1,3 +1,20 @@
...
 
-# Register your models here.
+from base.admin import TimeStampedAdmin
+
+from .models import (ChallengeHost,
+                     ChallengeHostTeam)
+
+
+@admin.register(ChallengeHostTeam)
+class HostTeamModelAdmin(TimeStampedAdmin):
```
Comment (deshraj) [inline apps/hosts/admin.py]: My bad. I forgot to comment here. Can you please replace `HostTeamModelAdmin` with `ChallengeHostTeamModelAdmin`?

Comment (gauthamzz) [inline apps/hosts/admin.py]: done :+1:

Comment (gauthamzz) [inline apps/hosts/admin.py]: @deshraj  is it okay now ?

**Comment by gauthamzz** [comment]
@deshraj sorry it took so long

**Comment by deshraj** [comment]
@gauthamzz No worries. Glad we merged it into master. :-)

**Comment by gauthamzz** [comment]
can you review #149  also

**Comment by deshraj** [comment]
Sure.

**Pull request merged**